### PR TITLE
adding home page

### PR DESCRIPTION
**Pull request by sanjay-xdr**
demo:https://objective-jackson-0fc355.netlify.app/index.html

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+            ></a>
+          </li>
+          <li>
+            <a
+              href="https://www.linkedin.com/in/sanjay-jaiswal-603757202/"
+              class="font-l anchor-style nav__item--hover footer__icon"
+              ><i class="fa fa-linkedin" aria-hidden="true"></i
+            ></a>
+          </li>
+          <!-- <li><a href="" class="font-l anchor-style"><i class="fa fa-linkedin" aria-hidden="true"></i></a></li> -->
```
Comment (tushargoyal812) [inline index.html]: remove comments

File: style.css [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+.moredeal-option {
+  background-color: transparent;
+  border: 0rem;
+}
+
+.footer {
+  height: 15rem;
+}
+
+.footer__icon {
```
Comment (tushargoyal812) [inline style.css:59]: use one format for naming like if you are using hyphen then use it everywhere.

**Review by tushargoyal812 — COMMENTED** [review]
minor changes and good to merge.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+      <div class="nav-sitename">
+        <h1><a href="#" class="nav-logo anchor">Guitar Store</a></h1>
+      </div>
+
+      <div class="nav__searchbar">
+        <input type="text" class="input" />
+        <button class="btn btn-secondary">Search</button>
+      </div>
+
+      <div class="nav-other">
```
Comment (SRUJANA-PENUGONDA13) [inline index.html:21]: You can name this class name in more descriptive

File: style.css [hunk 4]
```diff
@@ -0,0 +1,72 @@
+@import url(https://fsharpui.netlify.app/main.css);
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+:root{
+  --white-color:#fff;
```
Comment (SRUJANA-PENUGONDA13) [inline style.css]: We should not name colours like this. We are using variables for the purpose of reusability. Suppose if you want to change your particular colour of your background to blue that variable name that doesn't make sense. 

Ex- Your major color is gray then name it as primary color

**Review by SRUJANA-PENUGONDA13 — COMMENTED** [review]
I have suggested few changes once go through it. Reaming all good. Keep up good work

**Review by HrishikeshMahalle — APPROVED** [review]
Apart from basic errors, do note the above suggestions as well.

File: style.css [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+}
+
+:root{
+  --nav-color:#fff;
+  --grey-color:#808080;
+
+}
+.navigation {
+  margin: 0rem;
+  width: 100%;
```
Comment (HrishikeshMahalle) [inline style.css:15]: Zero can just be left with no values as well

**Pull request merged**